### ISSUE TITLE
option to startAt

### ISSUE THREAD

**Issue by adardesign** [NickPiscitelli/Glider.js#88]
Is there a option to `startAt` a certain slide?

**Comment by NickPiscitelli** [comment]
No but you could use the methods to slide to a slide immediately after
initialization.

**Comment by adardesign** [comment]
Thanks, can you please specify which exact methods?

**Comment by adardesign** [comment]
I tried `scrollItem`
Can you please look at https://codesandbox.io/s/mystifying-brook-fd19n

**Comment by NickPiscitelli** [comment]
Yes, scrollItem is the method to use. Call it after initialization.

**Comment by adardesign** [comment]
Thanks, can’t use that with the react wrapper

**Comment by NickPiscitelli** [comment]
I did not make the react wrapper so you might want to pose the question in
that repository, but there is no reason that you should not be able to call
the 'scrollItem' method on the glider object.

**Comment by clarencefoy** [comment]
Just to weigh in here, I think I'd be in favour of a 'startAt' function. Or, at the very least, a way to skip the scroll animation.
Right now, with a 10+ image gallery, it takes quite a while to scroll to the final image, if, for example, I am trying to open a full-screen gallery on the same image as a site-embedded one.

**Comment by dave-vdg** [comment]
Any updates on this? 
Is there a way to disable the animation so there is no weird scrolling when going to that view?
